docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.4"
+
+services:
+  localstack-s3:
+    image: localstack/localstack:latest
+    container_name: localstack-s3
```

Comment:
this shouldn't be needed in general, as the service name `localstack-s3` is used to refer to the container's hostname from any other container

Reply:
removed.